README.md

```diff
@@ -180,14 +180,22 @@ The MIC uses the service principal [credentials stored in the cluster] to access
 
 If the Azure identity is in the same resource group as your AKS cluster nodes, you can skip this section. (For [AKS], a resource group was added with an `MC_` prefix when you created the cluster.) Otherwise, follow these steps to assign the required permissions:
 
-1. Find the service principal used by your cluster. Please refer to the [AKS docs] for details. For example, if you didn't specify a service principal to the `az aks create` command, it will have generated one in the `~/.azure/aksServicePrincipal.json` file.
+1. Find the service principal used by your cluster. Please refer to the [AKS docs] for details. For example, if you didn't specify a service principal to the `az aks create` command, it will have generated one in the `~/.azure/aksServicePrincipal.json` file.You can also run this command:
```

Comment:
nit: can you please rephrase in the following way, so that one way is provided and any other ways can be obtained from the detailed documentation..

1. Find the service principal used by your cluster. You can also run the following command to obtain the SP:
```Bash
 az aks show -g <resourcegroup> -n <name> --query servicePrincipalProfile.clientId -o tsv
 ```
Replace the `<resourcegroup>` with the name of the resource group that holds the AKS Cluster and `<name>` with the name of the AKS Cluster.

For more information please refer to the [AKS docs].

Reply:
@spaelling - could you please update the PR with the changes so that we can go ahead and merge it.